lib/v2/ruijuzi/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'rui.juzi.bot': {
+        _name: '李佳芮博客',
+        www: [
+            {
+                title: '首页',
+                docs: 'https://docs.rsshub.app/blog.html#li-jia-rui-bo-ke',
+                source: ['/feeds.html'],
```

Comment:
```suggestion
                source: ['/'],
```
`https://rui.juzi.bot/feeds.html` does not exist